core/src/com/po/fuck/model/Constants.java

```diff
@@ -1,4 +1,4 @@
-package com.po.fuck;
```

Comment:
I am not sure that the are only in the model, because we have a lot of variable for sizes/etc that are not related to model

Reply:
Possibly split into several constants files.